### DISCUSSION TITLE
How many moons does Neptune have?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#58] [Galaxy Trivia] [answered]
**Hint::** 2 + 2 * 2

**Comment by Tynab** [comment, marked as answer]
Neptune has 14 known moons. Their names, in order of distance from Neptune, are Naiad, Thalassa, Despina, Galatea, Larissa, S/2004 N 1, Proteus, Triton, Nereid, Halimede, Sao, Laomedeia, Psamathe, and Neso. The largest of these is Triton, which is also the seventh largest moon in the solar system. It is believed that many of Neptune's moons were captured from the Kuiper Belt, a region of the solar system beyond the orbit of Neptune that is home to many icy objects.
